python/railjson_generator/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "railjson_generator"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
Why do you do this change in your PR ? Do you need it ?

Reply:
Change made with @bloussou